CONTRIBUTING.md

```diff
@@ -258,3 +258,16 @@ the main branch. The workflow file should be named as
 [![NuGet version badge](https://img.shields.io/nuget/v/{your_package_name})](https://www.nuget.org/packages/{your_package_name})
 [![NuGet download count badge](https://img.shields.io/nuget/dt/{your_package_name})](https://www.nuget.org/packages/{your_package_name})
 ```
+
+### Guidance for components on supporting target frameworks
+
+* SHOULD support the [.NET
```

Comment:
nit: add one more "MUST document the target frameworks supported in the main readme.md file".